.github/workflows/ci-repo.yml

```diff
@@ -61,14 +65,14 @@ jobs:
         run: npm run test -- --coverage --watchAll=false
 
       - name: Upload to codecov (client)
-        if: ${{ github.event.pull_request }}
+        # if: ${{ github.event.pull_request }}
```

Comment:
our PR branches are mostly short-lived - I'm not sure if this is the main reason of codecov being out of synch (as of this PR its 200+ commits). Let's see it in action.